class/defaults.yml

```diff
@@ -55,7 +55,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.92.6
+        tag: v4.92.8
```

Comment:
that's my fault, I tagged and pushed appcat automatically (muscle memory :D after merge), so I'm poinitng to the latest